CONTRIBUTING.md

```diff
@@ -44,8 +43,12 @@ src/:
 
 ## Testing
 
-- Write unit tests for each new function (if applicable)
-- Add integration tests for end-to-end scenarios
+- Feel free to write temporary unit tests during development if they help you verify functionality
+- These tests can be rough and don't need to be comprehensive - they're just development aids
```

Comment:
I really like the wording here, it feels welcoming and inclusive!

Reply:
we just need some instruction's/help for integration tests, those can be hard to write. This one https://github.com/RustAudio/rodio/blob/6f2518eb1f4ad4e542294f37c074e112f2df1502/tests/seek.rs#L119 took me a ton of time. We should not expect contributors to write *that* involved integration tests but it would be very nice if the could.

The wav test is a good example to point to I think: https://github.com/RustAudio/rodio/blob/master/tests/wav_test.rs

Reply:
took care of it, please take a look let me know what you think